modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -32,96 +37,96 @@ class {{classname}}(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        {{#vars}}'{{name}}': '{{{datatype}}}'{{#hasMore}},
-        {{/hasMore}}{{/vars}}
+{{#vars}}
+        '{{name}}': {{{datatype}}}{{#hasMore}},{{/hasMore}}
+{{/vars}}
     }
 
     attribute_map = {
-        {{#vars}}'{{name}}': '{{baseName}}'{{#hasMore}},
-        {{/hasMore}}{{/vars}}
+{{#vars}}
+        '{{name}}': '{{baseName}}'{{#hasMore}},{{/hasMore}}
+{{/vars}}
     }
 {{#discriminator}}
 
     discriminator_value_class_map = {
-        {{#children}}'{{vendorExtensions.x-discriminator-value}}': '{{{classname}}}'{{^-last}},
+        {{#children}}'{{^vendorExtensions.x-discriminator-value}}{{name}}{{/vendorExtensions.x-discriminator-value}}{{#vendorExtensions.x-discriminator-value}}{{{vendorExtensions.x-discriminator-value}}}{{/vendorExtensions.x-discriminator-value}}': '{{{classname}}}'{{^-last}},
```

Comment:
Resolves issue where if the user doesn't specify `x-discriminator-value` and the key value was always an empty string.